.gitignore

```diff
@@ -41,3 +41,4 @@ testem.log
 .DS_Store
 Thumbs.db
 /tmp
+/s4e-web/.vscode
```

Comment:
I'd rather edit the section, which is explicitly about VSCode, lines 14-18. It assumes you want to commit some settings and so on, and you probably don't want to do this.

Reply:
I was thinking more like leaving just this in the VSCode section:
```
# IDE - VSCode
.vscode/*
```
as I don't think anyone else depends on it in our team, and it would be cleaner.

Reply:
ok